clinvar/src/main/scala/org/broadinstitute/monster/etl/clinvar/models/intermediate/VariationArchive.scala

```diff
@@ -250,57 +247,109 @@ object VariationArchive {
           val scvId = rawScv
             .extract("@ID")
             .getOrElse {
-              throw new IllegalStateException(s"Found an SCV with no numeric ID: $scv")
+              throw new IllegalStateException(s"Found an SCV with no numeric ID: $rawScv")
             }
             .str
-          scvIdToAccession.update(scvId, scv.id)
+          scvIdToAccession.update(scvId, scvAccessionId)
           val relevantMappings = mappingsByScvId.getOrElse(scvId, Array.empty)
 
           rawScv.extract("TraitSet").foreach { rawTraitSet =>
-            val traitSet = SCVTraitSet.fromRawAssertionSet(scv, rawTraitSet)
             val traitCounter = new AtomicInteger(0)
+            val currentScvTraitIds = new mutable.ArrayBuffer[String]()
             MsgTransformations.popAsArray(rawTraitSet, "Trait").foreach { rawTrait =>
               val scvTrait = SCVTrait.fromRawTrait(
-                traitSet,
+                scvAccessionId, // the setId is the same as the scv.id when extracting from assertions
```

Comment:
Can you move the comment to the line above `scvAccessionId`? It keeps the line from sprawling too wide.